### DISCUSSION TITLE
Do we have an accessibility specialist? (Asking for Authy mobile resourcing)

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#2138] [Q&A] [answered]
Does the paste team have an accessibility specialist? Asking because the Authy app is talking about accessibility concerns, and it's come up in certain support tickets... so I was wondering if we have any accessibility resources or specialists we could talk to about how to start the journey to making the Authy app accessible. Thank you!

**Comment by SiTaggart** [comment, marked as answer]
Hi @mjanssen2021 

Though we don't have a dedicated "accessibility specialist" as a role on the team, we do have a number of subject-matter experts that are more than willing to help out and give advice.

@shleewhite and myself worked together on the Salesforce accessibility team for a number of years, and there are plenty of folks on the team with great knowledge on the subject from Design to Engineering. Our area of expertise is desktop web, though we probably know enough about mobile to get you mainly where you need to be.

Please sign up for an office hours slot and we can look at what you have.

**Reply by mjanssen2021** [reply]
Thanks Simon! Ill be sure and let my team know about this and we will attend an office hours soonish.
